packages/components/navbar/src/NavbarSwitcher/NavbarEnvVariant.tsx

```diff
@@ -12,19 +12,11 @@ export function NavbarEnvVariant({
   isAlias,
   envVariant,
 }: NavbarEnvVariantProps) {
-  const styles = getNavbarEnvVariantStyles();
+  const color = envVariant === 'master' ? tokens.green600 : tokens.orange500;
 
-  return (
-    <span
-      className={
-        envVariant === 'master' ? styles.envItemMaster : styles.envItemNonMaster
-      }
-    >
-      {isAlias ? (
-        <EnvironmentAliasIcon size="tiny" />
-      ) : (
-        <EnvironmentIcon size="tiny" />
-      )}
-    </span>
+  return isAlias ? (
+    <EnvironmentAliasIcon color={color} size="tiny" />
```

Comment:
are these not still old icons?

Reply:
No they are from the alpha: https://github.com/contentful/forma-36/blob/afc9f97d979e6e516c63b3f43d44348d70b629db/packages/components/navbar/package.json#L14